File: baekjoon/문자열/5622.py
result = 0

def getTime(alphabet):
  if alphabet in ('A','B','C'):
    return 3
  elif alphabet in ('D','E','F'):
    return 4
  elif alphabet in ('G','H','I'):
    return 5
  elif alphabet in ('J','K','L'):
    return 6
  elif alphabet in ('M','N','O'):
    return 7
  elif alphabet in ('P','Q','R','S'):
    return 8
  elif alphabet in ('T','U','V'):
    return 9
  elif alphabet in ('W','X','Y','Z'):
    return 10
  else:
    return 0 

for alphabet in input():
  result = result + getTime(alphabet)
print(result)

File: baekjoon/문자열/5622.js
const readline = require('readline')
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
})

let input = []

rl.on('line', (line) => {
  input.push(line)
}).on('close', () => {
  let result = 0
  for (alphabet of input[0]) {
    result += getTime(alphabet)
  }
  console.log(result)
  process.exit()
})

const getTime = (alphabet) => {
  switch (alphabet) {
    case 'A':
    case 'B':
    case 'C':
      return 3
    case 'D':
    case 'E':
    case 'F':
      return 4
    case 'G':
    case 'H':
    case 'I':
      return 5
    case 'J':
    case 'K':
    case 'L':
      return 6
    case 'M':
    case 'N':
    case 'O':
      return 7
    case 'P':
    case 'Q':
    case 'R':
    case 'S':
      return 8
    case 'T':
    case 'U':
    case 'V':
      return 9
    case 'W':
    case 'X':
    case 'Y':
    case 'Z':
      return 10

    default:
      return 0
  }
}
